src/sql/workbench/contrib/notebook/browser/notebookActions.ts

```diff
@@ -402,7 +404,7 @@ export class AttachToDropdown extends SelectBox {
 	private handleContextsChanged(showSelectConnection?: boolean) {
 		let kernelDisplayName: string = this.getKernelDisplayName();
 		if (kernelDisplayName) {
-			this.loadAttachToDropdown(this.model, kernelDisplayName, showSelectConnection);
+			this.loadAttachToDropdown(this.model, kernelDisplayName, showSelectConnection).catch(e => this.logService.error(e));
```

Comment:
Preferred to use onUnexpected error instead of log service, onUnexpectedError will handle logging it for us (and do so in a much prettier format)

Reply:
since logService is already being used in several places in this file, I just followed that. If this is a better way to do it, I can change it in other places as well.